Source/Fuse.LocalNotifications/Android/Impl.uno

```diff
@@ -68,7 +68,7 @@ namespace Fuse.LocalNotifications
                         String title = newIntent.getStringExtra("title");
                         String body = newIntent.getStringExtra("bbody");
                         String payload = newIntent.getStringExtra(@{ACTION});
-                        String result = "{ 'title': '" + title + "', 'body': '" + body + "', 'payload': '" + payload + "' }";
+						String result = com.fuse.LocalNotifications.LocalNotificationReceiver.MakePayloadString(title, body, payload);
```

Comment:
You're mixing indentation styles here. While you're using the one that the code-style mandates, This just gets hard to read. Could you either keep spaces, or start the PR with a commit that changes the indentation style for the whole file?

Reply:
I didnt do that because I remembered your hatred of whitespace change commits. Happy to do that though

Reply:
I'll just make it conform with current style